source/FilesServer/app/api.py

```diff
@@ -0,0 +1,72 @@
+import boto3
+from botocore.exceptions import ClientError
+from fastapi import Response, APIRouter, HTTPException, UploadFile, status
+from urllib.parse import quote
+import os
+
+
+# Check if the environment variables are set
+if (
+    not os.getenv("AWS_ACCESS_KEY_ID")
```

Comment:
use pydantic settgins for this. i this that if you don't specify a default value pydantic will throw an error if some of the values weren't provided an env var (need to be verified, i'm not sure)